spec/observables/interval-spec.ts

```diff
@@ -1,64 +1,101 @@
+/** @prettier */
 import { expect } from 'chai';
-import { expectObservable } from '../helpers/marble-testing';
 import { NEVER, interval, asapScheduler, animationFrameScheduler, queueScheduler } from 'rxjs';
 import { TestScheduler } from 'rxjs/testing';
 import { take, concat } from 'rxjs/operators';
 import * as sinon from 'sinon';
-
-declare const rxTestScheduler: TestScheduler;
+import { observableMatcher } from '../helpers/observableMatcher';
 
 /** @test {interval} */
 describe('interval', () => {
+  let rxTestScheduler: TestScheduler;
+
+  beforeEach(() => {
+    rxTestScheduler = new TestScheduler(observableMatcher);
+  });
+
   it('should create an observable emitting periodically', () => {
-    const e1 = interval(20, rxTestScheduler).pipe(
-      take(6), // make it actually finite, so it can be rendered
-      concat(NEVER) // but pretend it's infinite by not completing
-    );
-    const expected = '--a-b-c-d-e-f-';
-    const values = {
-      a: 0,
-      b: 1,
-      c: 2,
-      d: 3,
-      e: 4,
-      f: 5,
-    };
-    expectObservable(e1).toBe(expected, values);
+    rxTestScheduler.run(({ expectObservable, time }) => {
+      const values = {
+        a: 0,
+        b: 1,
+        c: 2,
+        d: 3,
+        e: 4,
+        f: 5,
+      };
+
+      const period = time('--|           ');
+      //                     --|
+      //                       --|
+      //                         --|
+      //                           --|
+      //                             --|
+      //                               --|
+      const expected = '   --a-b-c-d-e-f-';
+
+      const e1 = interval(period).pipe(
+        take(6), // make it actually finite, so it can be rendered
+        concat(NEVER) // but pretend it's infinite by not completing
+      );
+
+      expectObservable(e1).toBe(expected, values);
+    });
   });
```

Comment:
I'm not sure that we need this test at all. It seems that this test used to be used as a placeholder for the generation of the marble diagram images.

Since we don't create marble diagram images out of the tests anymore, comments `// make it actually finite, so it can be rendered` and `// but pretend it's infinite by not completing` are obsolete.

If the entire test does not make any value (or, if there is a test that does something similar), I would suggest removing it.

However, if the test makes any value, at least removal of these comments would be a good idea. And probably `concat(NEVER)` is not needed as well...

Reply:
I don't exactly know if it can be removed. I also guess, that it was used to render diagram images.

It seems that the test next test in the file asserts pretty much the same as this one.

Reply:
Yeah, many files contain very similar first and second test, just because the first one was used for marble rendering purposes. I suggest removing this test, or at least removing comments related to the marble diagrams. However, I'd like to hear @cartant and/or @benlesh's opinion.

Reply:
Yeah, I agree. Let's remove the test.